tasks/users.yaml

```
@@ -1,10 +1,12 @@
 ---
 - name: Create users and database grants
+  when: mariadb.galera.cluster == False or mariadb.galera.initializer == True
   mysql_user:
     name: "{{ username }}"
     password: "{{ password }}"
     host: "{{ host }}"
     priv: "{{ priv }}"
+    login_unix_socket: "{{ mariadb.socket if mariadb.socket else null }}"
```

Default is none
```suggestion
    login_unix_socket: "{{ mariadb.socket if mariadb.socket else none }}"
```
https://github.com/ansible/ansible/blob/feb163e0de503b83be4b0faf27cc378f2350ce70/test/support/integration/plugins/modules/mysql_user.py#L695
https://github.com/ansible/ansible/blob/feb163e0de503b83be4b0faf27cc378f2350ce70/lib/ansible/module_utils/basic.py#L1842-L1844

---

I wanted to use None, as it is standard in Python, but Ansible does not accept None. So that was the only one I knew of. I didn't know of none, thanks for pointing that out.